File: VB/RoleGenerator/CustomizeCodeLinesEventArg.vb
Imports DevExpress.Persistent.Base
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace RoleGeneratorSpace
    Public Class CustomizeCodeLinesEventArg
        Public ReadOnly Property Role As IPermissionPolicyRole
        Public Property CustomCodeLines As List(Of String)

        Public Sub New(ByVal role As IPermissionPolicyRole, ByVal customCodeLines As List(Of String))
            Me.Role = role
            Me.CustomCodeLines = customCodeLines
        End Sub
    End Class
End Namespace

File: CS/XPO/GenerateRole/RoleGenerator/CustomizeCodeLinesEventArg.cs
using DevExpress.Persistent.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace RoleGeneratorSpace
{
	public class CustomizeCodeLinesEventArg
	{
		public IPermissionPolicyRole Role { get; }
		public List<string> CustomCodeLines { get; set; }
		public CustomizeCodeLinesEventArg(IPermissionPolicyRole role, List<string> customCodeLines)
		{
			Role = role;
			CustomCodeLines = customCodeLines;
		}
	}
}
